client/web/src/enterprise/batches/detail/BatchChangeDetailsPage.tsx

```diff
@@ -131,15 +131,19 @@ export const BatchChangeDetailsPage: React.FunctionComponent<
                     />
                 }
                 actions={
-                    <BatchChangeDetailsActionSection
-                        batchChangeID={batchChange.id}
-                        batchChangeClosed={!!batchChange.closedAt}
-                        deleteBatchChange={deleteBatchChange}
-                        batchChangeNamespaceURL={batchChange.namespace.url}
-                        batchChangeURL={batchChange.url}
-                        history={history}
-                        settingsCascade={props.settingsCascade}
-                    />
+                    batchChange.viewerCanAdminister ? (
+                        <BatchChangeDetailsActionSection
+                            batchChangeID={batchChange.id}
+                            batchChangeClosed={!!batchChange.closedAt}
+                            deleteBatchChange={deleteBatchChange}
+                            batchChangeNamespaceURL={batchChange.namespace.url}
+                            batchChangeURL={batchChange.url}
+                            history={history}
+                            settingsCascade={props.settingsCascade}
+                        />
+                    ) : (
+                        <></>
+                    )
```

Comment:
```suggestion
                    ) : null
```

Minor nitpick but it might be better to render `null` here instead of an empty fragment so nothing shows up In the DOM.

Reply:
Good call; will update. :+1: